WatchDogEclipsePlugin/WatchDogUnitTests/src/nl/tudelft/watchdog/logic/event/EventPersisterBug101Test.java

```diff
@@ -0,0 +1,34 @@
+package nl.tudelft.watchdog.logic.event;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import nl.tudelft.watchdog.core.logic.event.EventPersisterBase;
+
+public class EventPersisterBug101Test extends EventPersisterTestBase {
```

Comment:
What is this test for and why does it exist for events (I don't think we have bug reports for event persisting yet, do we)?